rplugin/python3/molten/images.py

```diff
@@ -199,11 +199,68 @@ def remove_image(self, identifier: str) -> None:
         self.to_make_invisible.add(identifier)
 
 
+class WeztermCanvas(Canvas):
+    """A canvas for using Wezterm's imgcat functionality to render images/plots"""
+
+    nvim: Nvim
+
+    def __init__(self, nvim: Nvim):
+        self.nvim = nvim
+        self.initial_pane_id: int | None = None
+        self.image_pane: int | None = None
+
+    def init(self) -> None:
+        self.nvim.exec_lua("_wezterm = require('load_wezterm_nvim').wezterm_api")
+        self.wezterm_api = self.nvim.lua._wezterm
+        self.initial_pane_id = self.wezterm_api.get_current_pane()
+
+    def deinit(self) -> None:
+        """Closes the terminal split that was opened with MoltenInit"""
+        self.wezterm_api.close_image_pane(str(self.image_pane).strip())
+
+    def present(self) -> None:
```

Comment:
these functions should be doing something. There's not much point in extending the `Canvas` class if they're just empty.

There should be a way to fill these in and also remove the code that you wrote that sorta follows the image popup stuff. Image popup is special b/c it needs to exist alongside canvases. This should be it's own canvas.